docs/source/markdown/options/file.md

```diff
@@ -1,16 +1,31 @@
 ####> This option file is used in:
-####>   podman build, farm build
+####>   podman build, podman-build.unit.5.md.in, farm build
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
+<< if is_quadlet >>
+### `File=Containerfile`
+<< else >>
 #### **--file**, **-f**=*Containerfile*
+<< endif >>
+
 
 Specifies a Containerfile which contains instructions for building the image,
 either a local file or an **http** or **https** URL.  If more than one
 Containerfile is specified, *FROM* instructions are only be accepted from the
 last specified file.
 
+<< if is_quadlet >>
+Note that for a given relative path to a Containerfile, or when using a `http(s)://` URL, you also must set
```

Comment:
```suggestion
Note that for a given relative path to a Containerfile, or when using a `http(s)://` URL, also set
```
We try to avoid pronouns like you in man pages.